docs/plugin-development.rst

```diff
@@ -804,6 +804,27 @@ This demonstrates one simple use case for client plugins, but using these same
 techniques, you should be able to do almost anything to change the core
 application as you need.
 
+JavaScript events
+*****************
+
+The JavaScript client handles events from both the server and client-specific
+code.  The server events originate on the server and can be monitored by both
+the server's python code and the client's JavaScript code.  The client events
+are solely within the web client, and do not get transmitted to the server.
+
+If the connection to the server is interrupted, the client will not receive
+server events.  Periodically, the client will attempt to reconnect to the
+server to resume handling events.  Similarly, if client's browser tab is placed
+in the background for a long enough period of time, the connection that listens
+for server events will be stopped to prevent excessive resource use.  When the
+browser's tab regains focus, the client will once again receive server events.
+
+When the connection to the server's event stream is interrupted, a
+``g:eventStream.stop`` event is triggered.  When the server is once more
```

Comment:
We should clarify that ``g:eventStream.stop`` is a Backbone event, triggered on the `EventStream` object.